### PR TITLE
fix past event listing and typo

### LINKED ISSUE #1945: Error when clicking on link to expired event
When I click on the following link (to a valid, but expired event) I get a backend error:

https://app.couchers.org/event/138/couch-surfing-happy-hour

image

When I click on the link of an expired event, I should be able to view it even though it's expired.

### PR DESCRIPTION
**Pull request by lucaslcode**
fixes #1945
python sux

**Backend checklist**
- [x] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [x] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [x] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

### CODE REVIEW

File: app/backend/src/tests/test_events.py [hunk 1]
```diff
@@ -2033,3 +2037,40 @@ def test_can_overlap_other_events_update_regression(db):
...
+    # test for a bug where listing past events didn't work if they didn't have a future occurence
+    user, token = generate_user()
+
+    with session_scope() as session:
+        c_id = create_community(session, 0, 2, "Community", [user], [], None).id
+
+    start = now()
+
+    with events_session(token) as api:
+        # create another event, should be able to overlap with this one
```
Comment (kalvdans) [inline app/backend/src/tests/test_events.py]: Remove cut-and-pasted comment

File: app/backend/src/couchers/servicers/events.py [hunk 2]
```diff
@@ -771,8 +771,8 @@ def ListAllEvents(self, request, context):
                     EventOccurrence.start_time.desc()
                 )
 
-            occurances = occurrences.limit(page_size + 1)
-            occurrences = session.execute(occurances).scalars().all()
+            occurrences = occurrences.limit(page_size + 1)
```
Comment (aapeliv) [inline app/backend/src/couchers/servicers/events.py:774]: I am ashamed.

**Review by aapeliv — DISMISSED** [review]
Awesome! Thanks for fixing this up!!

**Review by kalvdans — DISMISSED** [review]
Yes, [first should really be named first_or_none](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.first)

**Review by aapeliv — APPROVED** [review]

**Review by kalvdans — APPROVED** [review]

**Pull request merged**